test/fixtures.js

```diff
@@ -38,3 +38,17 @@ export const RANGE_PASSWORD_CLEAN_RESPONSE = stripIndents`
   01B70E6A414C4E408CBFCE994104CEF2292:2
   01E2642ADE813D24A943A0AFA9218B543B7:1
 `;
+export const RANGE_PASSWORD_PWNED_RESPONSE_JSON = [
+  {
+    suffix: '003D68EB55068C33ACE09247EE4C639306B',
+    count: 3,
+  },
+  {
+    suffix: '1E4C9B93F3F0682250B6CF8331B7EE68FD8',
+    count: 3303003,
+  },
+  {
+    suffix: '01330C689E5D64F660D6947A93AD634EF8F',
+    count: 1,
+  },
+];
```

Comment:
Let's change all occurrences of `RANGE_PASSWORD_PWNED_RESPONSE_JSON` to something like `RANGE_PASSWORD_PWNED_RESPONSE_PARSED` (since it's not JSON).